### PR TITLE
Host catalog capabilities

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

**Review by randallmorey — APPROVED** [review]

File: ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs [hunk 1]
```diff
@@ -41,12 +41,14 @@
...
-                  {{hostCatalog.displayName}}
-                </LinkTo>
+                {{#if (can 'read model' hostCatalog)}}
+                  <LinkTo
+                    @route='scopes.scope.host-catalogs.host-catalog'
+                    @model={{hostCatalog.id}}
+                  >
+                    {{hostCatalog.displayName}}
+                  </LinkTo>
+                {{/if}}
```
Comment (randallmorey) [inline ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs:51]: Else clause?

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: ui/admin/app/components/form/host-catalog/index.hbs [hunk 2]
```diff
@@ -45,12 +45,12 @@
...
   </form.textarea>
-
-  <form.actions
-    @disabled={{if @model.cannotSave @model.cannotSave}}
-    @enableEditText={{t 'actions.edit-form'}}
-    @submitText={{t 'actions.save'}}
-    @cancelText={{t 'actions.cancel'}}
-  />
-
+  {{#if (can 'save model' @model)}}
```
Comment (calcaide) [inline ui/admin/app/components/form/host-catalog/index.hbs:48]: I am missing an else case and wondering if the user has no save capabilities, it just sees the form with no actions? Perhaps the else is not need here?

Comment (DhariniJeeva) [reply ui/admin/app/components/form/host-catalog/index.hbs:48]: oops, seems like my changes didn't get pushed. It needs an else case.

Comment (DhariniJeeva) [reply ui/admin/app/components/form/host-catalog/index.hbs:48]: yes, we don't need the else case here. The user can't edit the form if they don't have `update` capability.  `save model` is a composite that includes `update`.  see discussion [here](https://github.com/hashicorp/boundary-ui/pull/689)

Comment (calcaide) [reply ui/admin/app/components/form/host-catalog/index.hbs:48]: Nice! I miss that discussion

**Review by calcaide — APPROVED** [review]

**Review by randallmorey — APPROVED** [review]

**Pull request merged**